src/py/aspen/database/models/sample.py

```diff
@@ -48,6 +50,43 @@ class PhysicalSample(idbase):
         comment="This is the public identifier we assign to this sample.",
     )
 
+    sample_collected_by = Column(
```

Comment:
i feel like since this is already in the Sample table that we don't need these `sample_` prefixes to these columns?

Reply:
That's the way I had it originally.  @sidneymbell changed it, so maybe she can speak to her thinking on this matter?